cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -87,7 +88,7 @@
     private static final boolean DEFAULT_KAFKA_METRICS_ENABLED = false;
 
     // Kafka configuration defaults
-    private static final String DEFAULT_KAFKA_ZOOKEEPER_CONNECT = "zookeeper:2181";
+    private static final String DEFAULT_KAFKA_ZOOKEEPER_CONNECT = "zookeeper-client:2181";
```

Comment:
Taking a look at the code I was just wondering if this default makes sense to have. If Zookeeper is deployed using Strimzi, it's name will be always something line `xxx-zookeeper-client` and you now have changed the assignment as follow : 

`result.setZookeeperConnect(ZookeeperCluster.serviceName(kafkaAssembly.getMetadata().getName()) + ":2181");`

The default is really used just for this declaration : 

`private String zookeeperConnect = DEFAULT_KAFKA_ZOOKEEPER_CONNECT;`

which is overridden by above. Maybe we don't need it.

Reply:
Yeah, you are right ... in reality it is never really used. We always change the value.